azure/README-enterprise.md

```diff
@@ -30,10 +30,13 @@ kubectl create secret generic kubecost-thanos -n kubecost --from-file=object-sto
 # Create Cloud Integration Secret
 kubectl create secret generic cloud-integration -n kubecost --from-file=cloud-integration.json
 
-# Azure service key
+# Azure service key (optional)
```

Comment:
Quick comment about what the service key option turns on/what it limits?

Reply:
finding unused LBs and PVs from my understanding. also spot prices in azure?

Reply:
I can add that detail there, it is just spread out a bit